### DISCUSSION TITLE
Actualización Primefaces 12 a Primefaces 14. <p:colunm filterOptions=""/> no existe.

### DISCUSSION THREAD

**Discussion by igalpa** [primefaces/community#2403] [PrimeFaces] [answered]
He estado mirando los cambios de una versión a otra y no vi que desapareciera esta opción.
¿Ha desaparecido o tengo algo mal?
En Primefaces 12 lo tenia asi:
![image](https://github.com/user-attachments/assets/2857916c-832c-4c1a-89b5-f423fd13c870)
En Primefaces 14 no lo tengo.
![image](https://github.com/user-attachments/assets/ea3e28ae-166d-4726-93b9-1281c5c99256)
Y si lo pongo me indica:
![image](https://github.com/user-attachments/assets/6cc62196-e8b8-4950-96e9-bac2b7492764)
Podéis decirme si ha desaparecido o tengo algún error.
Gracias de antemano.

**Comment by melloware** [comment, marked as answer]
See: https://primefaces.github.io/primefaces/15_0_0/#/../migrationguide/13_0_0?id=deleted

> UIColumn#filterOptions removed (in favor of facet filter)

Instead use `<facet>` like https://primefaces.github.io/primefaces/15_0_0/#/components/datatable?id=filtering

```xml
<p:column field="manufacturer" headerText="Manufacturer" filterMatchMode="exact">
        <f:facet name="filter">
            <p:selectOneMenu onchange="PF('carsTable').filter()" styleClass="ui-custom-filter">
                <f:selectItem itemLabel="All" itemValue="#{null}" noSelectionOption="true" />
                <f:selectItems value="#{carBean.manufacturerOptions}" />
            </p:selectOneMenu>
        </f:facet>
    </p:column>
```
